### PR TITLE
orientdb-2.0.12

### PR DESCRIPTION
**Pull request by poindextrose**

### CODE REVIEW

File: Library/Formula/orientdb.rb [hunk 1]
```diff
@@ -3,9 +3,9 @@
 class Orientdb < Formula
   desc "Graph database"
   homepage 'http://www.orientdb.org/index.htm'
-  url 'http://www.orientechnologies.com/download.php?email=unknown@unknown.com&file=orientdb-community-2.0.11.tar.gz&os=mac'
-  version '2.0.11'
-  sha1 '812df4b971c3afc7bc9760b7de9bcc6680c060f0'
+  url 'http://www.orientechnologies.com/download.php?email=unknown@unknown.com&file=orientdb-community-2.0.12.tar.gz&os=mac'
```
Comment (vszakats) [inline Library/Formula/orientdb.rb]: I'd suggest this secure `url` that also follows permanent redirect and omits unnecessary parameters:
`url "https://orientdb.com/download.php?file=orientdb-community-2.0.12.tar.gz"`

File: Library/Formula/orientdb.rb [hunk 2]
```diff
@@ -3,9 +3,9 @@
 class Orientdb < Formula
   desc "Graph database"
   homepage 'http://www.orientdb.org/index.htm'
```
Comment (vszakats) [inline Library/Formula/orientdb.rb]: Suggesting a perm redirect and secure URL here:
`url "https://orientdb.com/"`

File: Library/Formula/orientdb.rb [hunk 3]
```diff
@@ -3,9 +3,9 @@
...
-  url 'http://www.orientechnologies.com/download.php?email=unknown@unknown.com&file=orientdb-community-2.0.11.tar.gz&os=mac'
-  version '2.0.11'
-  sha1 '812df4b971c3afc7bc9760b7de9bcc6680c060f0'
+  url 'http://www.orientechnologies.com/download.php?email=unknown@unknown.com&file=orientdb-community-2.0.12.tar.gz&os=mac'
+  version '2.0.12'
+  sha1 '23f180e89fa36e3bcba2c063d2987888ca095524'
```
Comment (DomT4) [inline Library/Formula/orientdb.rb]: `sha256` instead of SHA1 here would be :tada: as well.

File: Library/Formula/orientdb.rb (hunk 1 above)
Comment (DomT4) [inline Library/Formula/orientdb.rb]: ^ :+1:

File: Library/Formula/orientdb.rb [hunk 4]
```diff
@@ -3,9 +3,9 @@
 class Orientdb < Formula
   desc "Graph database"
   homepage 'http://www.orientdb.org/index.htm'
-  url 'http://www.orientechnologies.com/download.php?email=unknown@unknown.com&file=orientdb-community-2.0.11.tar.gz&os=mac'
-  version '2.0.11'
-  sha1 '812df4b971c3afc7bc9760b7de9bcc6680c060f0'
+  url 'https://orientdb.com/download.php?file=orientdb-community-2.0.12.tar.gz'
+  version '2.0.12'
```
Comment (DomT4) [inline Library/Formula/orientdb.rb]: You can remove this `version` line. The URL can be read by `brew` now.

**Comment by dunn** [comment]
@BrewTestBot test this please

**Comment by dunn** [comment]
Test failure might just be a bot hiccup, so if you want to update the URL like @vszakats suggests that'll kick off another build and we'll see if it fails again.

**Comment by vszakats** [comment]
👍🏻

**Comment by DomT4** [comment]
Presuming dead. Feel free to resubmit as a fresh PR with the various comments addressed if desired. Thanks for the submission!

**Pull request closed without merging**